core/scriptloader.js

```diff
@@ -114,10 +114,30 @@ CKEDITOR.scriptLoader = ( function() {
 
 					// Create the <script> element.
 					var script = new CKEDITOR.dom.element( 'script' );
-					script.setAttributes( {
-						type: 'text/javascript',
-						src: url
-					} );
+					if (self.trustedTypes && self.trustedTypes.createPolicy) {
+						const policy = self.trustedTypes.createPolicy(
+						  'scriptloader#loadScript',
+						  {
+							createScriptURL: function (url) {
+								const regexURL = /^https:\/\/example\.com(\/.*)?$/;
```

Comment:
is this supposed to be example.com?

Reply:
For now, we were planning to keep it as a placeholder example.com and add our internal regex in the build file unless there was some better way to keep a placeholder here?

Reply:
Changed this to a legacy conversion